.github/workflows/release.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: release
+
+jobs:
+  build:
+    name: tag release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env
+        run:  echo "RELEASE_TAG=${GITHUB_REF:10}" >> $GITHUB_ENV
+      - name: checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install go
+        uses: actions/setup-go@v2
```

Comment:
Interesting this is using a different go version than the cover.yaml

Reply:
cover.yaml is using the latest go version from https://golang.org/VERSION?m=text which is 1.17 now. For this I'm using 1.16 because that's the min version for needed for the release-notes tool.

I asked @alexeldeib why cover.yaml isn't using the setup-go action to install go and it just wasn't around when he first set this up so I think we could switch cover to also using the action instead of having our own script.

Reply:
bumped to go 1.17 and changed cover to also use the setup go action